src/Client.php

```diff
@@ -172,6 +172,11 @@ private function configureDefaults(array $config)
             $defaults['proxy']['https'] = $proxy;
         }
 
+        if ($noProxy = getenv('NO_PROXY')) {
+            $cleanedNoProxy = preg_replace('/\s/', '', $noProxy);
```

Comment:
It may be better to use trim function?
```
php > var_dump(trim(" \t "));
string(0) ""
```
If no why only spaces will be removed?